### ISSUE TITLE
Bug: zig fmt removes empty comments and collapses additional whitespace

### ISSUE THREAD

**Issue by MasterQ32** [ziglang/zig#7722]
When formatting the following snippet, it collapses to a unexpected result:

**original:**
```zig
comptime {
    a();

    // 
    b();  
}
```

**current output:**
```zig
comptime {
    a();
    b();  
}
```

**expected output:**
```zig
comptime {
    a();
    
    b();  
}
```

Is that wanted behaviour? When removing the empty comment, the empty line is preserved

**Comment by dbandstra** [comment]
Yeah this always bugged me too. I would prefer to keep the empty comment as well though.

**Comment by LemonBoy** [comment]
[Easy peasy](https://github.com/ziglang/zig/blob/79730e6f5cdf13b2514781881e69c557fd52eaea/lib/std/zig/render.zig#L2459)

**Comment by ifreund** [comment]
Fixed in #7920
